tests/playwright/sanity/assets/dev/js/editor/elements/views/widget.test.js

```diff
@@ -9,7 +9,9 @@ test.describe( `$e.run( 'editor/elements/views/widget' )`, () => {
 
 		// Act.
 		await editor.addWidget( 'image-carousel' );
-		await editor.getPreviewFrame().waitForSelector( '.elementor-widget .elementor-widget-empty-icon' );
+		await editor.getPreviewFrame().waitForSelector( '.elementor-widget-image-carousel .elementor-widget-empty-icon', {
+			state: 'visible',
```

Comment:
state: 'visible',

not required, this is the default

Reply:
https://github.com/elementor/elementor/pull/21559/commits/516fb74dfbd83b4de876f1c3bb1f4f8c63b72949